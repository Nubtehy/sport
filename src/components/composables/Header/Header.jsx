
import React from 'react';
import { HeaderWrapper, HeaderTitle } from './Header.styled';


const displayName = 'Header';

const propTypes = {
};

const defaultProps = {

};

function Header({title}) {
  return (
    <HeaderWrapper>
      <HeaderTitle>
        Sports store {title}
      </HeaderTitle>
    </HeaderWrapper>
  );
}


Header.displayName = displayName;
Header.propTypes = propTypes;
Header.defaultProps = defaultProps;

export default Header;
