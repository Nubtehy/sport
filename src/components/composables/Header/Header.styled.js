import styled from 'styled-components';

export const HeaderWrapper = styled.header`
  flex: 0 0 auto;
  padding: 20px;
  background-color: #cccccc;
`;

export const HeaderTitle = styled.h1`
  font-size: 30px;
`;
