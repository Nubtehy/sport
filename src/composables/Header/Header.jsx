
import React from 'react';


const displayName = 'Header';

const propTypes = {
};

const defaultProps = {

};

function Header() {
  return (
    <header>
      <div>
        This is header
      </div>
    </header>
  );
}


Header.displayName = displayName;
Header.propTypes = propTypes;
Header.defaultProps = defaultProps;

export default Header;
