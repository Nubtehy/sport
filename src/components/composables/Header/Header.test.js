import React from 'react';
import { shallow } from 'enzyme';

import Header from './Header';

describe('Given a Header component', () => {
  describe('when the component is rendered', () => {
    let component;

    beforeEach(() => {
      component = shallow(<Header />);
    });

    it('should match the snapshot', () => {
      expect(component).toMatchSnapshot();
    });
  });
});
