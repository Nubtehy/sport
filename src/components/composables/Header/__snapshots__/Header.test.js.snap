// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Given a Header component when the component is rendered should match the snapshot 1`] = `
<Headerstyled__HeaderWrapper>
  <Masterstyled__CartContainer>
    <Headerstyled__HeaderTitle>
      Sports store
       
    </Headerstyled__HeaderTitle>
  </Masterstyled__CartContainer>
</Headerstyled__HeaderWrapper>
`;
